src/scripts/client.js

```diff
@@ -2,26 +2,40 @@ const avengersNames = ['Thor', 'Cap', 'Tony Stark', 'Black Panther', 'Black Wido
 let randomName = avengersNames[Math.floor(Math.random() * avengersNames.length)];
 
 const main = async () => {
-    /* Events handlers */
-    VoxeetSDK.conference.on('streamAdded', (participant, stream) => {
-        if (stream.type === 'ScreenShare') return addScreenShareNode(stream);
-        addVideoNode(participant, stream);
-        addParticipantNode(participant);
-    });
-
-    VoxeetSDK.conference.on('streamRemoved', (participant, stream) => {
-        if (stream.type === 'ScreenShare') return removeScreenShareNode();
-        removeVideoNode(participant);
-        removeParticipantNode(participant);
-    });
-
-    try {
-        await VoxeetSDK.initialize('customerKey', 'customerSecret');
-        await VoxeetSDK.session.open({ name: randomName });
-        initUI();
-    } catch (e) {
-        alert('Something went wrong : ' + e);
+  /* Event handlers */
+
+  // When a video stream is added to the conference
+  VoxeetSDK.conference.on('streamAdded', (participant, stream) => {
+    if (stream.type === 'ScreenShare') {
+      return addScreenShareNode(stream);
+    }
+
+    addVideoNode(participant, stream);
+    addParticipantNode(participant);
+  });
+
+  // When a video stream is removed from the conference
+  VoxeetSDK.conference.on('streamRemoved', (participant, stream) => {
+    if (stream.type === 'ScreenShare') {
+      return removeScreenShareNode();
     }
+
+    removeVideoNode(participant);
+    removeParticipantNode(participant);
+  });
+
+  try {
+    // Initialize the Voxeet SDK
+    VoxeetSDK.initialize('customerKey', 'customerSecret');
```

Comment:
My preference would be to completely remove the reference to VoxeetSDK.initialize call and force user to try out the token authentication instead. Our getting started guide doc (https://dolby.io/developers/interactivity-apis/client-sdk/initializing) describes that paradigm and an example of the authentication server is available as part of the voxeet-io-web project. If we really want to keep it very simple, at least we should put a warning comment here and let user know this is not the recommended way of initializing the SDK.

Reply:
Agreed, this is not a best practice to use the `initialize`, I will add a warning and a link to the documentation page.